Another public Apple M1 Tech Preview (#12306)
docker/docker.github.io: docker-for-mac/apple-m1.md

    ```
    softwareupdate --install-rosetta
    ```
- The HTTP proxy is not enabled.
- Not all images are available for ARM64. You can add `--platform linux/amd64` to run an Intel image under emulation.

    In particular, the [mysql](https://hub.docker.com/_/mysql?tab=tags&page=1&ordering=last_updated){: target="blank" rel="noopener" class=“”} image is not available for ARM64. You can work around this issue by using a [mariadb](https://hub.docker.com/_/mariadb?tab=tags&page=1&ordering=last_updated){: target="blank" rel="noopener" class=“”} image.
- The kernel may panic. If so, look in `~/Library/Containers/com.docker.docker/Data/vms/0/console.log` for a BUG or kernel panic to report.

> ## Fixes since the last preview

Reviewer: ```suggestion
## Fixes since the Apple Silicon preview 7

2021-02-11
```

Reviewer: Specified the preview version and the date so users can track changes

- Kubernetes now works (although you might need to reset the cluster in our Troubleshoot menu one time to regenerate the certificates).
- osxfs file sharing works.
- The `host.docker.internal` and `vm.docker.internal` DNS entries now resolve.

> - We removed hard-coded IP addresses: it now dynamically discovers the IP allocated by macOS.

Reviewer: ```suggestion
- Removed hard-coded IP addresses: Docker Desktop now dynamically discovers the IP allocated by macOS.
```

> - We made a change that should improve disk performance.

Reviewer: ```suggestion
- The updated version includes a  change that should improve disk performance.
```
